indexer/services/comlink/src/lib/restrict-countries.ts

```diff
@@ -0,0 +1,42 @@
+import { stats } from '@dydxprotocol-indexer/base';
+import express from 'express';
+
+import config from '../config';
+import { create4xxResponse } from './helpers';
+import { isRestrictedCountry } from './utils';
+
+/**
+ * Return an error code for users that access the API from a restricted country
+ */
+export function rejectRestrictedCountries(
+  req: express.Request,
+  res: express.Response,
+  next: express.NextFunction,
+) {
+  const {
+    'cf-ipcountry': ipCountry,
+  } = req.headers as {
+    'cf-ipcountry'?: string,
+  };
```

Comment:
this type definition is super weird, im assuming because js doesn't want hyphens in a field?

Reply:
Correct.